pkg/analysis/helpers/markers/analyzer_test.go

```diff
@@ -0,0 +1,51 @@
+package markers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtractMarkerIdAndExpressions(t *testing.T) {
```

Comment:
Nit, existing tests are all written with either ginkgo and gomega, or just gomega as the assertion library, would prefer to keep one assertion library at the minimum

Would look something like

```
g := NewWithT(t)

g.Expect(id).To(Equal(tc.expectedID))
g.Expect(expressions).To(Equal(tc.expectedExpressions))
```

assert.Equal(t, tc.expectedId, id, "marker", tc.marker)
		assert.Equal(t, tc.expectedExpressions, expressions, "marker", tc.marker)